golang-job-type/job-template.Dockerfile

```diff
@@ -14,6 +14,11 @@ COPY "{{ manifest.golang.gomod }}" /src/job/
 RUN cd /src/job && go mod download
 {% endif %}
 
+{% if manifest.jobtype_extra.gomod %}
```

Comment:
I guess it's better to use `manifest.get_jobtype_extra()`, cause I'm afraid the above-mentioned `if manifest.golang.gomod` might raise an error if `manifest.golang` is empty.
Plus, it won't duplicate the code.